homeassistant/components/default_config/__init__.py

```diff
@@ -18,6 +20,9 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if not is_hassio(hass):
         await async_setup_component(hass, "backup", config)
 
+    if sys.platform.startswith(("linux", "win32", "darwin")):
```

Comment:
These are all the platforms Home Assistant supports; therefore, this check isn't needed.

Reply:
You asked me to fix the issue myself. I did. What the hell is your problem? This hurts nobody.